packages/bot/src/components/save-load-modal.jsx

```diff
@@ -0,0 +1,185 @@
+                                        text={translate('Google Drive')}
+                                        google_drive_connected={isGoogleDriveConnected}
+                                        onDriveConnect={onDriveConnect}
+                                    />
+                                </div> */}
+                                {
+                                    is_save_modal ?
+                                        <>
+                                            <Checkbox
+                                                onChange={() => setValues({
+                                                    save_as_collection: !save_as_collection,
+                                                    is_local,
+                                                })}
+                                                defaultChecked={save_as_collection}
+                                                label={translate('Save as collection')}
+                                                classNameLabel='saveload-type__checkbox-text'
+                                            />
+                                            <div className='saveload-type__checkbox-description'>
+                                                {translate('A collection allow you to save block as an individual part which can be add into other bot')}
+                                            </div>
+                                        </> :
+                                        <input type='file' id='files' style={{ display: 'none' }} onChange={handleFileChange} />
+                                }
+                            </div>
+                            <div className='modal--footer'>
+                                <Button
+                                    type='button'
+                                    className={classNames(
+                                        'save-load__button',
+                                        'btn--secondary',
+                                        'btn--secondary--orange',
+                                    )}
+                                    text={translate('Cancel')}
+                                    onClick={() => toggleSaveLoadModal(is_save_modal)}
+                                />
+                                <Button
+                                    type='submit'
+                                    className={classNames(
+                                        'save-load__button',
+                                        'btn--primary',
+                                        'btn--primary--orange',
+                                    )}
+                                    text={translate('Continue')}
+                                />
+                            </div>
+                        </Form>
+                    )
+                }
+            </Formik>
+        </Modal>
+    );
+};
+
+// const IconCheckbox = props => {
```

Comment:
can add comment here on what it will do in the future, or TODO when need to be removed

Reply:
This is be on my next PR, I'm already doing it now. 😂